authors/settings.py

```diff
@@ -80,8 +80,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```

Comment:
You can have defaults here. Basically for local development. @e-ian

Reply:
I have included the defaults @inno-asiimwe